quantum/process_keycode/process_unicode_common.c

```diff
@@ -243,6 +243,25 @@ void send_unicode_string(const char *str) {
     }
 }
 
+void register_unicode(uint32_t code_point, uint8_t input_mode) {
```

Comment:
`send_unicode_string()` should also call this function, I guess?

Reply:
Yep, that would make sense! I also just realized that the `input_mode` parameter of this function is redundant, since Unicode functions always use the globally set input mode.

@Str8AWay I've opened another PR on your branch, Str8AWay#2, where I've made these changes, and also updated this branch with the latest changes from the QMK master.

Reply:
Merged your updates in